purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -668,8 +669,121 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     }
 
     // endregion
+    // region Attribution IDs
+
+    fun collectDeviceIdentifiers() {
+        executeOnBackground {
```

Comment:
Add log

Reply:
there might not be a need for `Purchases` to know that this method needs to be called on the background, maybe moving that bit into the code that collects identifiers would simplify it